### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by StevenWafeek**
In this project we : 
- Add new HTML elements 
- Add new CSS elements
- Add Javascript  to the project
- Fix readme file

### CODE REVIEW

File: javascript.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const nav = document.querySelector('.header-menu');
+const myClose = document.querySelector('.close');
+const myLinks = document.querySelector('.links');
+
+myOpen.onclick = () => {
+  nav.classList.add('open');
+  myOpen.classList.add('display');
+};
+myClose.onclick = () => {
+  this.parentElement.classList.remove('open');
```
Comment (didierganthier) [inline javascript.js]: Nice work on the popup 💯 but this line is throwing an error, seems like `parentElement` here is undefined, which is making it impossible to get the `classList`, which is also causing your menu not to close/disappear when the ❌ button is clicked.

File: javascript.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  myOpen.classList.remove('display');
+};
+myLinks.onclick = () => {
+  this.parentElement.classList.remove('open');
+  myOpen.classList.remove('display');
+};
+const MidLink = document.getElementById('about');
+MidLink.addEventListener('click', () => {
+  this.parentElement.classList.remove('open');
```
Comment (didierganthier) [inline javascript.js]: You did great work displaying the correct part of the page 👍🏽 but same as above, seems like `parentElement` here is undefined, which is making it impossible to get the `classList` and edit it to make the menu disappear. And as required 🤓  when the user clicks (or taps) the close (❌ ) button, the mobile menu should disappear. 😶‍🌫️

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi Team,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- N/A

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Mobile menu working as expected :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**